### ISSUE TITLE
Run 3: panicked at 'called `Result::unwrap()` on an `Err` value: "WebGL: Error in renderbuffer_storage_multisample (color): 1282"', render/webgl/src/lib.rs:739:35

### ISSUE THREAD

**Issue by Trung0246** [ruffle-rs/ruffle#1605]
![image](https://user-images.githubusercontent.com/11626920/99751923-1c1d2600-2a98-11eb-81a5-c0cb14adabfa.png)

Did not test Run and Run 2 yet (assuming you guys know these game already 😄), but seems like Run 3 have a weird problem that can crash the entire engine upon uploading the file (I used https://ruffle.rs/demo/ for testing)

[Run3_swf.txt](https://github.com/ruffle-rs/ruffle/files/5570959/Run3_swf.txt)

Uploaded the .swf sample as .txt (since github did not allow to upload .swf)

I could try to decompile the game to see what's going on but I don't have time to do that yet, probably later.

**Comment by BHSPitMonkey** [comment]
I encountered the same crash in Firefox (ruffle_nightly_2020_11_20_firefox.xpi) when zooming into the page (using Ctrl+Scroll Up) while viewing/interacting with [this page](https://homestarrunner.com/main13.html). Pasting the error output below:

---

# Error Info
Error name: Error
Error message: panicked at 'called `Result::unwrap()` on an `Err` value: "WebGL: Error in renderbuffer_storage_multisample (color): 1286"', render/webgl/src/lib.rs:739:35
Error stack:
```
$@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:20276
__wbg_new_df6e6ab7a65c4c4d@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/ruffle.js:1:31347
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[1201]:0x22757d
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[1358]:0x2436eb
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3589]:0x2eee47
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[2813]:0x2d15fd
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3884]:0x2f3a65
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3913]:0x2f3f9a
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3789]:0x2f2642
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3914]:0x2f3fd0
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3320]:0x2e8669
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[2842]:0x2d2e7e
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[766]:0x1b9a40
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3816]:0x2f2c1d
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[4075]:0x2f5b37
P@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17500
f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17376
FrameRequestCallback*699/Re<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:22481
q/<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:19028
__wbg_requestAnimationFrame_89935c9d6ac25d2f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/ruffle.js:1:35297
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3594]:0x2eefa8
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[766]:0x1b9bc2
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3816]:0x2f2c1d
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[4075]:0x2f5b37
P@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17500
f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17376
FrameRequestCallback*699/Re<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:22481
q/<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:19028
__wbg_requestAnimationFrame_89935c9d6ac25d2f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/ruffle.js:1:35297
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3594]:0x2eefa8
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[766]:0x1b9bc2
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3816]:0x2f2c1d
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[4075]:0x2f5b37
P@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17500
f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17376
FrameRequestCallback*699/Re<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:22481
q/<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:19028
__wbg_requestAnimationFrame_89935c9d6ac25d2f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/ruffle.js:1:35297
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3594]:0x2eefa8
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[766]:0x1b9bc2
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3816]:0x2f2c1d
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[4075]:0x2f5b37
P@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17500
f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17376
FrameRequestCallback*699/Re<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:22481
q/<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:19028
__wbg_requestAnimationFrame_89935c9d6ac25d2f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/ruffle.js:1:35297
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3594]:0x2eefa8
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[766]:0x1b9bc2
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3816]:0x2f2c1d
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[4075]:0x2f5b37
P@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17500
f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17376
FrameRequestCallback*699/Re<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:22481
q/<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:19028
__wbg_requestAnimationFrame_89935c9d6ac25d2f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/ruffle.js:1:35297
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3594]:0x2eefa8
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[766]:0x1b9bc2
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[3816]:0x2f2c1d
@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/85f5e4a29d83ee43a391.module.wasm:wasm-function[4075]:0x2f5b37
P@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17500
f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:17376
FrameRequestCallback*699/Re<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:22481
q/<@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/core.ruffle.js:1:19028
__wbg_requestAnimationFrame_89935c9d6ac25d2f@moz-extension://26dd1e0b-a397-4b47-adbd-b3a2ab0e9eed/dist/ruffle.js:1:35297

```

# Player Info
Allows script access: true
Player type: Object
SWF URL: main13.swf
Param FlashVars: weekliesCacheBust=1
Param movie: main13.swf
Param quality: high
Param bgcolor: #FFFFFF
Attribute 0: [object Attr]
Attribute 1: [object Attr]
Attribute 2: [object Attr]
Attribute 3: [object Attr]

# Page Info
Page URL: https://homestarrunner.com/main13.html

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:82.0) Gecko/20100101 Firefox/82.0
OS: Win32

# Ruffle Info
Ruffle version: 0.1.0
Ruffle source: undefined

**Comment by Herschel** [comment]
Fixed by #1767.
